### PR TITLE
Fix podman kube generate --help to show correct help message

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@nicrowe00 @vrothberg @edsantiago PTAL

**Comment by edsantiago** [comment]
lint is unhappy
```
cmd/podman/kube/generate.go:102:6: `kubeGenerate` is unused (deadcode)
```

File: test/system/260-kube.bats [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+function teardown() {
+    basic_teardown
+}
+
+@test "podman kube generate - basic" {
+    run_podman kube generate --help
+    is "$output" ".*podman.* kube generate \[options\] {CONTAINER...|POD...|VOLUME...}"
+    run_podman generate kube --help
+    is "$output" ".*podman.* generate kube \[options\] {CONTAINER...|POD...|VOLUME...}"
+}
```
Comment (nicrowe00) [inline test/system/260-kube.bats:23]: I would suggest moving this test to `test/system/700-play.bats` as this is where all the other `kube` related tests are currently located.

**Comment by nicrowe00** [comment]
Other than that small nit, LGTM.

**Review by edsantiago — COMMENTED** [review]
I'm deeply disturbed that we have *zero* tests for kube-generate or generate-kube! Now on my TODO list, so I'm OK if you merge this because I'll be making fixes soon anyway.

**Review by vrothberg — COMMENTED** [review]
Just a drive-by-comment asking for commit messages. Without reading the changes, it's hard to understand the problem.

**Pull request merged**